components/inputs/test/input-date.visual-diff.js

```diff
@@ -315,55 +297,74 @@ describe('d2l-input-date', () => {
 			});
 		});
 
-		describe('with placeholder', () => {
-			afterEach(async() => {
-				await reset(page, '#placeholder');
+		describe('with empty-text', () => {
```

Comment:
These were added because the state is different between open with enter vs with click and I wanted to capture that